template/web/gatsby-config.js

```diff
@@ -1,25 +1,20 @@
-// Load variables from `.env` as soon as possible
-require('dotenv').config({
-  path: `.env.${process.env.NODE_ENV || 'development'}`
-})
-
-const clientConfig = require('./client-config')
 const token = process.env.SANITY_READ_TOKEN
 
 const isProd = process.env.NODE_ENV === 'production'
+const previewEnabled = (process.env.GATSBY_IS_PREVIEW || "false").toLowerCase() === "true"
 
 module.exports = {
   plugins: [
     'gatsby-plugin-postcss',
     'gatsby-plugin-react-helmet',
+    'gatsby-plugin-image',
     {
       resolve: 'gatsby-source-sanity',
       options: {
         ...clientConfig.sanity,
         token,
         watchMode: !isProd,
-        overlayDrafts: !isProd && token
+        overlayDrafts: !isProd || previewEnabled,
```

Comment:
do we still need to check if `token` exists here, or is that just old code not really doing anything

Reply:
I used the recommended config in the source plugin readme, but I think that would make sense. If there is no token included, the preview breaks for drafted content.